Sources/SpecialErrorPages/SSLErrorType.swift

```diff
@@ -17,34 +17,32 @@
 //
 
 import Foundation
+import WebKit
 
-public enum SSLErrorType: String {
+public let SSLErrorCodeKey = "_kCFStreamErrorCodeKey"
+
+public enum SSLErrorType: String, Encodable {
 
     case expired
-    case wrongHost
     case selfSigned
+    case wrongHost
     case invalid
 
-    public static func forErrorCode(_ errorCode: Int) -> Self {
-        switch Int32(errorCode) {
-        case errSSLCertExpired:
-            return .expired
-        case errSSLHostNameMismatch:
-            return .wrongHost
-        case errSSLXCertChainInvalid:
-            return .selfSigned
-        default:
-            return .invalid
+    init(errorCode: Int) {
+        self = switch Int32(errorCode) {
+        case errSSLCertExpired: .expired
+        case errSSLXCertChainInvalid: .selfSigned
+        case errSSLHostNameMismatch: .wrongHost
+        default: .invalid
         }
     }
 
-    public var rawParameter: String {
```

Comment:
We send this rawParameter as parameter in the Pixels. Are we using the rawValue of the enum now? If so the value for "self_signed” will be different.